snapcraft.yaml.in

```diff
@@ -1,34 +0,0 @@
-name: ubuntu-personal-store
-version: "0.1"
-summary: A beautiful and functional interface for snap management.
-description: This application allows you to browse and install snap packages on your Ubuntu Personal device.
-
-grade: devel
-confinement: devmode
```

Comment:
Please make sure that the snapcraft,yaml that replaces this one has "confinement: devmode" too